src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -911,6 +923,47 @@ public void muxingFinished(final String streamId, File file, long startTime, lon
 
 	}
 
+	public void notifyRoomHook(String action, String roomId, String myTrackId, boolean playOnly) {
```

Comment:
Please remove the room concept. It's about mainTrack and subTracts. It can be used for multiview streaming or it can be used in conferencing or something else. 

I mean change the naming concept from room to tracks